lib/compiler-cranelift/src/translator/func_translator.rs

```diff
@@ -75,23 +72,26 @@ impl FuncTranslator {
     pub fn translate_from_reader<FE: FuncEnvironment + ?Sized>(
         &mut self,
         module_translation_state: &ModuleTranslationState,
-        reader: &mut dyn FunctionBinaryReader,
+        reader: &mut wasmer_compiler::FunctionReader,
         func: &mut ir::Function,
         environ: &mut FE,
     ) -> WasmResult<()> {
         let _tt = timing::wasm_translate_function();
-        info!(
-            "translate({} bytes, {}{})",
-            reader.bytes_remaining(),
-            func.name,
-            func.signature
-        );
+        let _span = tracing::info_span!(
```

Comment:
I wonder if we should rip out the `timing` module and just use spans? Still not sure about general applicability of tracing for profiling -- rust-analyzer still uses hand-rolled infra.

Reply:
The timing module is `cranelift`'s, not part of this codebase. Will definitely be punting on either decision for the time being. Unlike R-A this project is intended to be embedded, so making it use something off-shelf has significant benefits.